doc/integrator/docker.rst

```diff
@@ -97,6 +97,8 @@ Geoportal:
  * ``SQL_LOG_LEVEL``: Log level for the Sqlalchemy engine``, default is ``WARN``.
  * ``DOGPILECACHE_LOG_LEVEL``: Log level for Dogpile cache, default is ``INFO``.
  * ``OTHER_LOG_LEVEL``: Log level for other modules, default is ``WARN``.
+ * ``C2CGEOPORTAL_THEME_TIMEOUT``: Timeout for the theme generation, used on requests to get information
```

Comment:
Timeout in seconds used in requests sent to OGC servers during themes generation, default to ``300``.